### DISCUSSION TITLE
Change Layout Option for Android App

### DISCUSSION THREAD

**Discussion by LearningAsIGo71** [ente-io/ente#1030] [Enhancements]
As was suggested for web layout in issue #108, I would like to see a tile/grid type display implementation for the Android app much like how some launchers allow users to custom configure the amount of rows and columns for the Android home screen and/or app drawers. On my P7 phone I would prefer something like a 3 x 5 grid, but if I was using a tablet I would probably opt for 6 x 8. 

In the Ente Auth settings an "Appearance" category could be added and the Theme category moved into it along with a "Code Display" category which would allow users to choose between List (as it currently is) and Grid. 

With grid tiles there likely wouldn't be enough room for the "Next" code display, but I personally would be fine with that. Or you could add an option to long press a tile to display the Next code. 

Keep up the excellent work! I am very grateful for this app and to finally get away from awful Authy!

**Comment by AlphaElwedritsch** [comment]
Yes, I would also prefer a grid view

**Reply by NeeL21YT** [reply]
Bumping it!

**Comment by chloevo** [comment]
I would like a grid view with icons (with appname and username below the icon), and when you tap on the icon it will show full information on the top half of the screen.

**Comment by 0xAl3xH** [comment]
+1

**Comment by ghost** [comment]
Related: #4365

**Comment by kreintjes** [comment]
> I would like a grid view with icons (with appname and username below the icon), and when you tap on the icon it will show full information on the top half of the screen.

Yes this would be great! This sounds [similar to what Authy has](https://play-lh.googleusercontent.com/N_Ty4vWZl5ob-Oo0LXLyDQIRi2BVLwVJPD3q6csJWoD0-ZW6lPI5q3yc3iWvJ6KB9Pk=w5120-h2880). I really liked the grid view they offered with the icons. It makes it easy to quickly find a certain app.

And for the people who prefer a list, it would be great if it would be possible to switch between list and grid view (e.g. in settings or just directly from the main screen).

**Comment by kernelpanic-root** [comment]
Hope,we can get tile / Grid options.
